.github/workflows/release.yaml

```diff
@@ -3,90 +3,50 @@ on:
   push:
     tags:
       - "v*"
-env:
-  GO_VERSION: "1.18"
-  GH_USER: "aqua-bot"
+
 jobs:
   release:
-    name: Release
+    name: release
+    uses: ./.github/workflows/build.yaml
+    with:
+      goreleaser_file: goreleaser.yml
+      goreleaser_options: '--rm-dist --timeout 60m'
+    secrets: inherit
+
+  deploy-rpm-deb-packages:
+    name: deploy rpm/dep
+    needs: release # run this jobs release job completes
     runs-on: ubuntu-18.04 # 20.04 doesn't provide createrepo for now
-    env:
-      DOCKER_CLI_EXPERIMENTAL: "enabled"
-    permissions:
-      id-token: write # For cosign
-      packages: write # For GHCR
-      contents: read  # Not required for public repositories, but for clarity
     steps:
+      - name: Restore Trivy binaries from cache
+        uses: actions/cache@v3.0.2
+        with:
+          path: dist/
+          key: ${{ runner.os }}-bins-${{github.workflow}}-${{github.sha}}
+
       - name: Install dependencies
         run: |
           sudo apt-get -y update
           sudo apt-get -y install rpm reprepro createrepo distro-info
-      - uses: sigstore/cosign-installer@536b37ec5d5b543420bdfd9b744c5965bd4d8730
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@v2
-      - name: Set up Docker Buildx
-        id: buildx
-        uses: docker/setup-buildx-action@v2
-      - name: Show available Docker Buildx platforms
-        run: echo ${{ steps.buildx.outputs.platforms }}
-      - name: Setup Go
-        uses: actions/setup-go@v3
-        with:
-          go-version: ${{ env.GO_VERSION }}
-      - name: Checkout code
-        uses: actions/checkout@v3
-        with:
-          fetch-depth: 0
-      - name: Cache Go modules
-        uses: actions/cache@v3.0.2
-        with:
-          path: ~/go/pkg/mod
-          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
-          restore-keys: |
-            ${{ runner.os }}-go-
-      - name: Login to docker.io registry
-        uses: docker/login-action@v2
-        with:
-          username: ${{ secrets.DOCKERHUB_USER }}
-          password: ${{ secrets.DOCKERHUB_TOKEN }}
-      - name: Login to ghcr.io registry
-        uses: docker/login-action@v2
-        with:
-          registry: ghcr.io
-          username: ${{ env.GH_USER }}
-          password: ${{ secrets.GITHUB_TOKEN }}
-      - name: Login to ECR
-        uses: docker/login-action@v2
-        with:
-          registry: public.ecr.aws
-          username: ${{ secrets.ECR_ACCESS_KEY_ID }}
-          password: ${{ secrets.ECR_SECRET_ACCESS_KEY }}
-      - name: Generate SBOM
-        uses: CycloneDX/gh-gomod-generate-sbom@v1
-        with:
-          args: mod -licenses -json -output bom.json
-          version: ^v1
-      - name: Release
-        uses: goreleaser/goreleaser-action@v3
-        with:
-          version: v1.4.1
-          args: release --rm-dist --timeout 60m
-        env:
-          GITHUB_TOKEN: ${{ secrets.ORG_REPO_TOKEN }}
+
```

Comment:
Did this workflow work? I thought we needed to checkout Trivy. Otherwise, ci/deploy-rpm.sh doesn't exist.

Reply:
Could you test it in your forked trivy-repo? We may miss something else.

Reply:
I already checked this: https://github.com/DmitriyLewen/trivy/runs/7089906348?check_suite_focus=true
I forgot to add `GPG_KEY`. But `deploy-rpm.sh` worked correctly, so `deploy-deb.sh` will also work correctly.

Also successfully running canary build: https://github.com/DmitriyLewen/trivy/actions/runs/2575174982

Reply:
Could you import a dummy GPG key you generate and make sure it works in your fork?

Reply:
i checked. It works correctly : https://github.com/DmitriyLewen/trivy/actions/runs/2582486781